<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTrain</class>
 <widget class="QWidget" name="AddTrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>311</width>
     <height>256</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_0_i"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_1_n"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_2_m"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_3_s"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_4_p"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_5_x"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_6_t"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_7_o"/>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_8_d"/>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEdit_9_y"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3_n">
      <property name="text">
       <string>seatNum</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2_p">
      <property name="text">
       <string>stationNum</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5_g">
      <property name="text">
       <string>prices</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_1_u">
      <property name="text">
       <string>trainID</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4_m">
      <property name="text">
       <string>stations</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>startTime</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>travelTimes</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>stopoverTimes</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>saleDate</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>type</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="do_add_train">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
